README.md

```diff
@@ -1,3 +1,65 @@
+## Project title
```

Comment:
Love the effort you put into making a coherent README format. 

Keep doing that because it means [you're helping teams succeed, with code that is easy to change.](https://github.com/makersacademy/course/blob/master/goals/course_goals.md#i-help-my-teams-succeed)